VAMobile/src/screens/BenefitsScreen/ClaimsScreen/ClaimsHistoryScreen/ClaimsHistoryScreen.test.tsx

```diff
@@ -208,7 +209,7 @@ context('ClaimsHistoryScreen', () => {
       })
 
       expect(testInstance.findAllByType(AlertBox).length).toEqual(1)
-      expect(testInstance.findAllByType(TextView)[5].props.children).toEqual('Claims status is unavailable')
```

Comment:
I know we were redoing brittle unit tests as we encountered issues, but this ticket has already had way too many hang-ups so just did the quick fix.